### PR TITLE
E0d/maintainer reports

### PR DESCRIPTION
**Pull request by e0d**
Scripts for importing GitHub data in to BigQuery.

### CODE REVIEW

File: maintainer_reports/graphql_requests.py [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+    new_schema.append(bigquery.SchemaField("maintainer_organization", "STRING"))
+
+    table.schema = new_schema
+    table = client.update_table(table, ["schema"])
+
+
+def update_maintainers_bq(target_table):
+    """
+    Populated the maintainer related columns by joining against
+    an external table which is a Google Sheet.
```
Comment (feanil) [inline maintainer_reports/graphql_requests.py:160]: Should we link to this sheet in the Readme?

**Review by feanil — APPROVED** [review]
Generally looks good.  If this readme is meant to serve as the starting point for finding the infra for this as well, I recommend more links to places even if they would be behind login walls.

File: maintainer_reports/graphql_requests.py (hunk 1 above)
Comment (e0d) [reply maintainer_reports/graphql_requests.py:160]: Done.

File: maintainer_reports/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+A CLI for querying GraphQL data from GitHub and persisiting it to BigQuery for
```
Comment (nedbat) [inline maintainer_reports/README.md:1]: Are we gradually allowing .md instead of .rst ?

Comment (e0d) [reply maintainer_reports/README.md:1]: No, the standard is rst for anything that is part of the Open edX platform.  In this case capturing basic details quickly was prioritized.

**Pull request merged**